tests/gccgo/task.yaml

```diff
@@ -0,0 +1,13 @@
+summary: Check that snapd builds with gccgo
+prepare: |
+    echo Installing gccgo-6 and pretending it is the default go
+    sudo apt install -y gccgo-6
+    sudo ln -s /usr/bin/go-6 /usr/local/bin/go
+restore: |
+    rm -f /usr/local/bin/go
+    sudo apt-get autoremove -y gccgo-6
+execute: |
+    echo Build with gccgo with tests
+    pwd
```

Comment:
This can probably be removed. The remote pwd is always the directory of the task.yaml itself.